test/jasmine/tests/download_test.js

```diff
@@ -137,8 +137,8 @@ describe('Plotly.downloadImage', function() {
         })
         .then(done, done.fail);
     }, LONG_TIMEOUT_INTERVAL);
-
-    it('should produce right output in Safari', function(done) {
+/*
+    it('should produce right output in Old Safari', function(done) {
```

Comment:
Do we really want to keep these commented-out tests around? Unless you want to try and mock `canUseSaveLink===false` - which I don't really think is worth it at this point - I'd just delete them.